lib/multigrid.cpp

```diff
@@ -88,8 +88,9 @@ namespace quda
             }
           }
           if (param.mg_global.num_setup_iter[param.level] > 0) {
-            if (strcmp(param.mg_global.vec_infile[param.level], "")
-                != 0) { // only load if infile is defined and not computing
+            if (param.mg_global.vec_load[param.level] == QUDA_BOOLEAN_TRUE
```

Comment:
Was this a bug fix?

Reply:
Technically, though there was no code path where it was a problem in practice: `*.vec_load[*]` was actually never checked, and it was ultimately redundant of `*.vec_infile[*]` being non-null.

Previous behavior was by default `*.vec_load[*]` was unset, and was only set to `QUDA_BOOLEAN_TRUE` if `*.vec_infile[*]`was set to something non-null. This PR changes the behavior to initialize `*.vec_load[*]` to `QUDA_BOOLEAN_FALSE` and only set it to true as appropriate.

Another option is to just remove `*.vec_load[*]`, again because it's redundant of `*.vec_infile[*]` being non-null.